testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -52,6 +52,12 @@ public enum WhitelistLogLines {
                     "quarkus.oidc-client.discovery-enabled|" +
                     "quarkus.smallrye-jwt.enabled|" +
                     "quarkus.devservices.enabled|" +
+                    "quarkus.otel.traces.exporter|" +
```

Comment:
These entries are needed because of the LangChain4J set of extensions?

I would consider adding more "traditional" Quarkus extensions into `langchain4jExtensions` group, otel and hibernate could be good candidates (they would also invalidate need for these additions) GraphQL extension could get into the mix too.